Britannia/localization/english/BTWK_buildings_l_english.yml

```diff
@@ -39,78 +49,84 @@
- building_btwk_copper_mines_01:0 ""
- building_btwk_copper_mines_01_desc:0 ""
- building_btwk_copper_mines_02:0 ""
- building_btwk_copper_mines_02_desc:0 ""
- building_btwk_copper_mines_03:0 ""
- building_btwk_copper_mines_03_desc:0 ""
- building_btwk_copper_mines_04:0 ""
- building_btwk_copper_mines_04_desc:0 ""
+ building_btwk_copper_mines_01:0 "$building_btwk_mines_01$"
+ building_btwk_copper_mines_01_desc:0 "$building_btwk_mines_01_desc$"
+ building_btwk_copper_mines_02:0 "$building_btwk_mines_02$"
+ building_btwk_copper_mines_02_desc:0 "$building_btwk_mines_02_desc$"
+ building_btwk_copper_mines_03:0 "$building_btwk_mines_03$"
+ building_btwk_copper_mines_03_desc:0 "$building_btwk_mines_03_desc$"
+ building_btwk_copper_mines_04:0 "$building_btwk_mines_04$"
+ building_btwk_copper_mines_04_desc:0 "$building_btwk_mines_04_desc$"
 
  building_type_btwk_silver_mines_01:0 "Silver Mines" 
  building_type_btwk_silver_mines_01_desc:0 ""
- building_btwk_silver_mines_01:0 ""
- building_btwk_silver_mines_01_desc:0 ""
- building_btwk_silver_mines_02:0 ""
- building_btwk_silver_mines_02_desc:0 ""
- building_btwk_silver_mines_03:0 ""
- building_btwk_silver_mines_03_desc:0 ""
- building_btwk_silver_mines_04:0 ""
- building_btwk_silver_mines_04_desc:0 ""
+ building_btwk_silver_mines_01:0 "$building_btwk_mines_01$"
+ building_btwk_silver_mines_01_desc:0 "$building_btwk_mines_01_desc$"
+ building_btwk_silver_mines_02:0 "$building_btwk_mines_02$"
+ building_btwk_silver_mines_02_desc:0 "$building_btwk_mines_02_desc$"
+ building_btwk_silver_mines_03:0 "$building_btwk_mines_03$"
+ building_btwk_silver_mines_03_desc:0 "$building_btwk_mines_03_desc$"
+ building_btwk_silver_mines_04:0 "$building_btwk_mines_04$"
+ building_btwk_silver_mines_04_desc:0 "$building_btwk_mines_04_desc$"
 
  building_type_btwk_gold_mines_01:0 "Gold Mines"
- building_type_btwk_gold_mines_01_desc:0 "This area is rich with one of history's most coveted, precious metals - gold. While the metal is of course valuable, it was mostly used in the crafting of high value decorative pieces for those who could afford them. It was not, contrary to popular belief, typically used as a standard base for currency - not for over a millennia yet, at least."
- building_btwk_gold_mines_01:0 ""
- building_btwk_gold_mines_01_desc:0 ""
- building_btwk_gold_mines_02:0 ""
- building_btwk_gold_mines_02_desc:0 ""
- building_btwk_gold_mines_03:0 ""
- building_btwk_gold_mines_03_desc:0 ""
- building_btwk_gold_mines_04:0 ""
- building_btwk_gold_mines_04_desc:0 ""
+ building_type_btwk_gold_mines_01_desc:0 "This area is rich with one of history's most coveted, precious metals - gold. While the metal is of course valuable, it was mostly used in the crafting of high value decorative pieces for those who could afford them. During this period, a shift was made to using predominantly silver as the standard for currency in Europe - a change which would not be reverted until a over a millennium later."
```

Comment:
I think we don't really want to be directly referencing developments outside our time period - feels kind of out of context here

Reply:
Ties into same issue with Tor localisation - wasn't sure on the frame of reference they were being described from. Change it to be exclusively period specific?